packages/sources/metalsapi/src/endpoint/latest.ts

```diff
@@ -61,13 +61,19 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
-  const from = validator.overrideSymbol(AdapterName)
+  const base = validator.overrideSymbol(AdapterName)
   const to = validator.validated.data.quote
   const url = `latest`
+  if (Array.isArray(base))
+    throw new AdapterError({
+      jobRunID,
+      message: `Base symbol ${base} is not batchable.`,
+      statusCode: 400,
+    })
```

Comment:
I assume we'd never run into this under normal operation since we have the following set:
```typescript
export const batchablePropertyPath = [{ name: 'quote' }]
```

Am I understanding it right? cc @justinkaseman

Reply:
Yes, that is how it previously worked. I'll dig deeper into what is happening.

Reply:
Ok thanks. Btw this was not what broke it today, I think this is just adding more input validation.

Reply:
Ah no we should probably keep it. I just wanted to confirm that `batchablePropertyPath` would prevent us from sending an invalid request payload.